### DISCUSSION TITLE
User and group for Vector

### DISCUSSION THREAD

**Discussion by safin-na** [vectordotdev/vector#21907] [Q&A]
Hi, I'm in the process of rebuilding a server that had Vector on it. The service requires a Vector user and a Vector group to run. How do I add them? OS: Ubuntu 22.

**Comment by pront** [comment]
Hello, how did you install Vector? See documentation here: https://vector.dev/docs/administration/management/#linux

**Comment by jszwedko** [comment]
You might find the Debian preinst and postinst scripts useful as references:

- https://github.com/vectordotdev/vector/blob/master/distribution/debian/scripts/preinst
- https://github.com/vectordotdev/vector/blob/master/distribution/debian/scripts/postinst
